src/main/native/include/sysid/view/AnalyzerPlot.h

```diff
@@ -49,17 +49,22 @@ class AnalyzerPlot {
   /**
    * Displays voltage-domain plots.
    */
-  void DisplayVoltageDomainPlots();
+  void DisplayVoltageDomainPlots(bool forPicture = false);
 
   /**
    * Displays time-domain plots.
    */
-  void DisplayTimeDomainPlots();
+  void DisplayTimeDomainPlots(bool forPicture = false);
+
+  static constexpr int GetScreenshotPlotSize() { return kScreenshotPlotSize; }
 
  private:
   // The maximum size of each vector (dataset to plot).
   static constexpr size_t kMaxSize = 2048;
 
+  // Size of plots when screenshotting
+  static constexpr int kScreenshotPlotSize = 300;
```

Comment:
This can be public, and no need for a getter method. Rename to `kCombinedPlotSize`.